src/js/components/Offer/index.jsx

```diff
@@ -5,8 +5,18 @@ import {Link} from "react-router-dom";
 import Reputation from '../Reputation';
 import Identicon from "../UserInformation/Identicon";
 
-const Offer = ({offer, withDetail}) => (
-  <Row className="border bg-white rounded p-2 mr-0 ml-0 mb-2" tag={Link} to={`/profile/${offer.owner}`}>
+import './index.scss';
+
+function truncateTwo(num) {   
+  var re = new RegExp('^-?\\d+(?:\.\\d{0,' + (2 || -1) + '})?');
+  num = Number(num.toString().match(re)[0]);
+  return Number(Math.round(num + "e+2")  + "e-2");
+}
+
+const Offer = ({offer, withDetail, prices}) => {
+  const price = prices[offer.token.symbol][offer.currency];
+  const calcPrice = offer.marketType === "0" ? price * ((100 + (parseFloat(offer.margin))) / 100) : price * parseFloat(offer.margin) / 100;
```

Comment:
Lik I said in the other, might be better off as a utils

Reply:
I'll extract this as utils in the other PR since it's more recent!